pkgs/by-name/ka/kanidm/generic.nix

```diff
@@ -49,16 +64,13 @@ rustPlatform.buildRustPackage rec {
     inherit hash;
   };
```

Comment:
Do you think it would make sense to use `finalAttrs.version` in the rev? It kind of feels like it would be more correct (to me) but it doesn't actually make overriding any easier since overriding the hash requires overriding the src anyway.

Otherwise LGTM!

Reply:
Yes, but I'll plan on addressing that on next bump to avoid another round of building things.